packages/@o3r/create/src/index.ts

```diff
@@ -265,6 +266,21 @@ const prepareWorkspace = (relativeDirectory = '.', projectPackageManager = 'npm'
     }
   }
 
+  if (argv.minimal) {
```

Comment:
Not sure to understand this option.
If it is needed, you should also provide documentation regarding it in readme.
(and potentially tests :D)

Reply:
It is already documented [here]( https://github.com/AmadeusITGroup/otter/blob/main/packages/%40o3r/create/README.md#others-available-options) whne you go on this link https://v17.angular.io/cli/new#options

Reply:
In their case they are not generating the `.editorconfig` if they have the option `minimal` set to true

Reply:
> It is already documented [here](https://github.com/AmadeusITGroup/otter/blob/main/packages/%40o3r/create/README.md?rgh-link-date=2024-12-20T06%3A39%3A53Z#others-available-options), you can go on this link [v17.angular.io/cli/new#options](https://v17.angular.io/cli/new#options)

This is not really a fare answer :D.
The option itself is not documented but, as part of Angular option, is supported.
I think you can add a comment (with a link potentially) in the code regarding it.